react/features/base/jwt/functions.ts

```diff
@@ -76,23 +76,23 @@ interface IIsJwtFeatureEnabledStatelessParams {
  * @param {ILocalParticipant} localParticipantFeatures - The features of the local participant.
  * @param {string} feature - The feature we want to check.
  * @param {boolean} ifNoToken - Default value if there is no token.
- * @param {boolean} ifNotInFeatures - Default value if features prop exists but does not have the {@code feature}.
- * @returns {bolean}
+ * @param {boolean} ifNotInFeatures - Default value if features is missing
+ * or prop exists but does not have the {@code feature}.
+ * @returns {boolean}
  */
 export function isJwtFeatureEnabledStateless({
     jwt,
     localParticipantFeatures: features,
     feature,
-    ifNoToken = false,
-    ifNotInFeatures = false
+    ifNoToken,
+    ifNotInFeatures
 }: IIsJwtFeatureEnabledStatelessParams) {
     if (!jwt) {
         return ifNoToken;
     }
 
-    // If `features` is undefined, act as if everything is enabled.
     if (typeof features === 'undefined') {
-        return true;
+        return ifNoToken;
```

Comment:
I'm wondering shouldn't we return `ifNotInFeatures` in this use case?

Reply:
Well when there are no features, we want to respect isModerator, which we pass to ifNoToken.